src/components/tab-controls/tab-controls.scss

```diff
@@ -33,21 +33,18 @@
 		border-color: currentColor;
 	}
 
-	:checked + &:hover {
-		border-color: $color-focus-state;
-	}
-
-	.tab-controls__input:focus + & {
-		border-color: currentColor;
-		color: $color-focus-state;
-	}
-
 	&:hover {
-		color: $color-focus-state;
+		border-color: currentColor;
 		cursor: pointer;
 	}
 
-	h3 {
+	.tab-controls__heading {
 		margin-bottom: 0;
 	}
 }
+
+.tab-controls__heading {
```

Comment:
👍

Reply:
current: 
```scss
.tab-controls__input {

  // stuff 

	.tab-controls__heading {
		margin-bottom: 0;
	}
}

.tab-controls__heading {
	font-size: $textsize-m;
	font-weight: 500;
	line-height: 1.1;
}
```

updated:
```scss
.tab-controls__heading,
.tab-controls__input h3 {
	font-size: $textsize-m;
	font-weight: 500;
	line-height: 1.1;
  margin-bottom: 0;
}
```

@JosephCurley thoughts?

Reply:
I think that's a good idea, at least for now.